pkg/clusteragent/admission/controllers/webhook/controller_v1.go

```diff
@@ -256,3 +258,16 @@ func (c *ControllerV1) getAdmiV1FailurePolicy() admiv1.FailurePolicyType {
 		return admiv1.Ignore
 	}
 }
+
+func (c *ControllerV1) getReinvocationPolicy() admiv1.ReinvocationPolicyType {
+	policy := strings.ToLower(c.config.getReinvocationPolicy())
+	switch policy {
+	case "ifneeded":
```

Comment:
should we have an enum?

Reply:
in the [documentation](https://github.com/DataDog/datadog-agent/pull/11440/files#diff-a77843a3a2d99bdc21000210a1c6a0fc99871774b33d15b6779389e8299aa794R2444-R2450) it use `IfNeeded` instead

Reply:
Resolved (discussed offline)